MIP/mip-74/README.md

```diff
@@ -0,0 +1,165 @@
+
+1. the value of assets being transferred across the Native Bridge, within a configurable time window $\Delta$, MUST always be less than the insurance funds.
+1. the Governance Operator MUST be able to adjust the rate limit on the source and target chain.
+1. the Relayer MUST be able to catch up with the transfers in case it has been down for some time.
+1. the Relayer MAY adjust the rate limit on the source chain.
+
+The guiding principles of the design of the Rate Limiter are:
+
+1. the Governance Operator monitors the Native Bridge, and in case of an attack or fault, it SHOULD take at most $\Delta$ time units to detect the issue and pause the bridge.
+2. we want to make sure that the total amount that is transferred within $\Delta$ time units (and that could potentially result from  malicious behaviors) is ALWAYS covered by the insurance fund.
+
+### Rate Limiter
+
+The Rate Limiter limits the volume of assets that can be transferred within a time window. The Rate Limiter MUST be implemented as part of the L1 Native Bridge contract and the L2 Native Bridge contract.
+
+#### Insurance funds
+
+ We assume there is a Insurance Fund on both L1 and L2, with values `insurance_fund_L1` and `insurance_fund_L2`, respectively.
+
+> [!NOTE]
+> These values are considered constant in the sequel. There may be updated if needed or if new funds are added to the pools.
+
+The Insurance Fund rate-limits the transfers i.e., for a given transfer from source chain to target chain the Insurance Fund on the _target chain_ is responsible for the rate limit, and thus we will refer to the `insurance_fund_target`.
+For a transfer from L1 (L2) to L2 (L1) the `insurance_fund_target = insurance_fund_L2` (`insurance_fund_L1`) is responsible for the rate limit.
+
+#### Rate limit on the target chain
+
+The rate limit is dependent on the fund size in the Insurance Fund. In particular the maximum rate limit is defined by
+
+`max_rate_limit_target = insurance_fund_target / reaction_time`,
+
+where the `reaction_time` is the time it takes for the Governance Operator to react to a faulty or compromised component. The `reaction_time` is a parameter that is set by the Governance Operator. The Governance Operator MAY set the actual rate limit lower than the `max_rate_limit_target`. However the Rate Limiter MUST NOT set the rate limit higher than the `max_rate_limit_target`.
+
+The rate limit MAY also be adjusted by the Governance Operator.
+
+`rate_limit_target = rate_reduction_target * max_rate_limit_target`,
+
+where `rate_reduction_target` $\in$ `[0,1]` is a parameter that is set by the Governance Operator. Note the `rate_limit_target` MUST not be larger than `max_rate_limit_target`.
+
+The following are possible ways to adjust the rate limit:
+
+1. The Governance Operator can adjust the rate limit by adding or removing funds from the Insurance Fund.
+1. The Governance Operator may adjust the rate limit by changing the `reaction_time`.
```

Comment:
Current implementation is 24hrs and I think that a fixed, 24hrs `reaction_time` is fine. There are gas cost and logic issues with allowing the change of `reaction_time` that we should consider so that we don't bloat the contract too much.

Reply:
I have added this default parameter setting 

> _Implementation recommendation #1_: The default value is 24h. In the initial implementation this value is fixed to avoid complications in gas.